src/nidmm/metadata/config.py

```diff
@@ -15,10 +15,8 @@
             '64bit': {'name': 'libnidmm.so', 'type': 'cdll'},
         },
     },
-    'context_manager': [
-        {
-            'direction': 'input'
-        },
-    ]
+    'context_manager_name': {
+        'session': 'acquisition',
```

Comment:
I think 'session' is the wrong key. We have a Session class, and _it is_ a context manager.
This is something else. Best I can come up with is "task".

Reply:
This is the name of the object returned by the session context manager. Shouldn't that then be session?

Reply:
No, you're not the one confused. I am.

Changed to task.